### ISSUE TITLE
model: Catboost: Add model catboostregressor and catboostclassifier

### ISSUE THREAD

**Issue by nitesh585** [intel/dffml#970]
Catboost - https://github.com/catboost/catboost
To add model in dffml - https://intel.github.io/dffml/master/tutorials/models/index.html

**Comment by rajpratyush** [comment]
Can I work on this issue?

**Comment by pdxjohnny** [comment]
@rajpratyush Yes! (See https://intel.github.io/dffml/master/contributing/dev_community.html#communication-style)

**Comment by aryanxk02** [comment]
@nitesh585 hello, where in the tutorial page is it mentioned on adding a new model in dffml module?

**Comment by aryanxk02** [comment]
okay got it @nitesh585

**Comment by Saanidhyavats** [comment]
I want to contribute on this issue and will be creating a pr within 2 days.
